index.html

```diff
@@ -95,16 +95,16 @@ <h2 class="about">About myself</h2>
       <div class="formsection">
         <h3 class="styleh3">CONTACT US</h3>
         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-        <form action="" class="formstyle">
-          <input type="text" placeholder="Text input">
-          <input type="email" placeholder="email">
-          <textarea name="" id="" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <form action="https://formspree.io/f/moqrojze" method="POST" class="formstyle">
+          <input type="text" placeholder="Text input" maxlength="30" required>
+          <input type="email" placeholder="email" required>
+          <textarea name="text" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
+          <button type="submit" class="button_submit"><span>Submit</span></button>
         </form>
-        <button type="button" class="button_pro"><span>See Project</span></button>
         <p class="division_sections"></p>
       </div>
     </section>
-    <footer class="works">
+    <footer class="footer_class">
       <div class="logos">
         <img src="img/Vector.png" alt="">
         <img src="img/dos.png" alt="">
```

Comment:
Great job team 🤝

- [ ] [Optional] As stated in the project requirements, your project is supposed to replicate the Figma design as much as possible. You may please add the black line to the bottom of your design
![figma underline](https://user-images.githubusercontent.com/63917725/177393861-a2f631be-3769-48ad-9761-ffef89e1a2bb.PNG)